quantum/snippets.js

```diff
@@ -87,6 +87,25 @@ const CIRCUIT_DIAGRAM =
 `qc.draw(output='mpl')
 `;
 
+const GROVERS_ALGO =
+`from qiskit import QuantumCircuit
+from qiskit import Aer
+from qiskit.utils import QuantumInstance
+from qiskit.quantum_info import Statevector
+from qiskit.algorithms import Grover, AmplificationProblem
```

Comment:
```suggestion
from qiskit import Aer
from qiskit.quantum_info import Statevector
from qiskit.algorithms import Grover, AmplificationProblem
```
`QuantumCircuit` and `QuantumInstance` imports are redundant here.

Reply:
I think we might need them since this node is not necessarily used in the circuit.

Reply:
What are they needed for, given that the imports aren't used in this snippet?

Reply:
Oops, turns out they aren't really used. Thanks.